loops-and-arrays.js

```diff
@@ -42,6 +57,11 @@ const numbers = [2, 4, 5, 29, 38];
 
 const squares = [30, 2, 8, 24, 11];
 
+for (let i = 0; i < squares.length; i++) {
+    let volume = squares[i] * squares[i] * squares[i];
```

Comment:
volgens de uitwerkingen moet dit "const" zijn i.p.v. "let" - ik denk omdat de getallen in de array individueel niet aangepast worden, ze worden alleen met elkaar vermenigvuldigd.